src/components/ToastContainer.tsx

```diff
@@ -27,4 +39,17 @@ export const ToastContainer = ({ toasts, onDismissToast, inline = false, classNa
       {...toast}
       >{toast.message}</Toast>)}
   </StyledToastContainer>
-}
+};
+
+export const BodyPortalToastContainer = ({ toasts, onDismissToast, inline = false, className }: {
+  toasts: ToastData[], onDismissToast?: ToastData['onDismiss'], inline?: boolean, className?: string
```

Comment:
Can we make both of these a type/interface just so we don't forget these should take the same props

Reply:
Yeah, good idea